example/z-docker/README.md

```diff
@@ -59,28 +60,28 @@ Host my-droplet
 
 SSH into your droplet:
 
-```
+```bash
```

Comment:
Can you remove all these syntax settings? The GitHub bash syntax highighting unfortunately is quite poor, and it just makes a few random strings and pipe characters stick out while highlighting nothing else.

Reply:
Done